packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -136,6 +138,7 @@ export const ChatListItem = ({ chat, myId, isSelected, isRead }: IMgtChatListIte
   // if chat changes, update the internal state to match
   useEffect(() => {
     setChatInternal(chat);
+    checkWhetherToMarkAsRead(chat);
```

Comment:
It might be worth moving this...
`(chatInternal.id && (!chatInternal.chatType || !chatInternal.members))`

...into a isLoaded or isNotLoaded function or something so that you could check it here as well. There wouldn't be a lastUpdatedDateTime (line 168) if the chat isn't loaded which seems like it would cause an exception.